### ISSUE TITLE
Anti-missile defences

### ISSUE THREAD

**Issue by drNovikov** [zrisher/GardenWarfare#12]
I did some research today and found a turret that is good against missiles but rather weak against ships: http://steamcommunity.com/sharedfiles/filedetails/?id=294617824

I'd suggest tewaking this mod -- limiting the range to 1200 meters and probably disabling targeting anything else than missiles (if possible).

This defence is not impenetrable, but not as helpless against missile barrages as the regular gatlings. Also, more compact.

**Comment by zrisher** [comment]
I think there's definitely a place for laser-based missle countering turrets, but the primary choice for that will probably be the 20mm gatling turrets mentioned on our weapons list in the readme:
https://docs.google.com/spreadsheets/d/1IRxN2XdvChIX2UxErsPFg4dg2oq3L5k2btLZK1TCjPg/edit#gid=851124846

Specifically, see this:
https://en.wikipedia.org/wiki/Counter_Rocket,_Artillery,_and_Mortar
For the CRAM and Phalanx systems, perfect examples of how 20mm gatling turrets can rock missiles/rockets.

We can offer a laser-based version that offers the efficiency of just using power instead of ammunition and running silent. But it will need to be a more "advanced" block to build. The problem with shooting down missiles with lasers is that you need to transfer a good deal of heat over a very short time frame, because they move so fast. So you need good tracking and an intense beat. E.g.
https://www.youtube.com/watch?v=StC9nRB_AVY

**Comment by drNovikov** [comment]
I agree on the 20 mm CRAM, and I even have tested a mod yesterday that was inspired by the Phalanx. Unfortunately, that mod is OP against small ships while the laser turned out to be more effective against missiles while not wrecking small ships.
